container-search/src/test/java/com/yahoo/search/handler/test/SearchHandlerTestCase.java

```diff
@@ -23,19 +23,17 @@
 import com.yahoo.search.result.Hit;
 import com.yahoo.search.searchchain.Execution;
 import com.yahoo.search.searchchain.config.test.SearchChainConfigurerTestCase;
+import org.json.JSONObject;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
-import java.io.ByteArrayOutputStream;
-import java.io.File;
-import java.io.IOException;
+import java.io.*;
```

Comment:
More (or less!) wildcards.

Reply:
Configure IntelliJ to not use wildcard imports. They should not be used because then adding a type in the imported package may break your code.